cogs/utility.py

```diff
@@ -1396,46 +1460,82 @@ async def permissions_remove(
-                    f"current permission level is {perm.name}.",
-                )
-            else:
-                logger.info("Restored command permission level for `%s`.", name)
-                self.bot.config["override_command_level"].pop(name)
-                await self.bot.config.update()
-                perm = self.bot.command_perm(name)
+            name = name.split(" ")
+            type_ = name[0]
+            name = name[1]
+            if type_ not in {"command", "group"}:
+                return await ctx.send_help(ctx.command)
+            if type_ == "command":
+                extension = ctx.kwargs["name_and_role_or_user"]
+                if extension is not None:
+                    name += f" {extension}"
+                name = name.lower()
+                name = getattr(self.bot.get_command(name), "qualified_name", name)
+                level = self.bot.config["override_command_level"].get(name)
+                if level is None:
+                    perm = self.bot.command_perm(name)
+                    embed = discord.Embed(
+                        title="Error",
+                        color=self.bot.error_color,
+                        description=f"The command permission level was never overridden: `{name}`, "
+                        f"current permission level is {perm.name}.",
+                    )
+                else:
+                    logger.info("Restored command permission level for `%s`.", name)
+                    self.bot.config["override_command_level"].pop(name)
+                    await self.bot.config.update()
+                    perm = self.bot.command_perm(name)
+                    embed = discord.Embed(
+                        title="Success",
+                        color=self.bot.main_color,
+                        description=f"Command permission level for `{name}` was successfully restored to {perm.name}.",
+                    )
+                return await ctx.send(embed=embed)
+            if type_ == "group":
+                group = self.bot.get_cog(name.capitalize())
+                if group is None:
+                    embed = discord.Embed(
+                        title="Error",
+                        color=self.bot.error_color,
+                        description=f"The referenced {type_} does not exist: `{name}`.",
+                    )
+                    return await ctx.send(embed=embed)
+                for command in set(group.walk_commands()):
+                    name = command.qualified_name
+                    level = self.bot.config["override_command_level"].get(name)
+                    if level is None:
+                        continue
+                    else:
+                        logger.info("Restored command permission level for `%s`.", name)
+                        self.bot.config["override_command_level"].pop(name)
+                        await self.bot.config.update()
```

Comment:
```suggestion
                update = False
                for command in set(group.walk_commands()):
                    name = command.qualified_name
                    level = self.bot.config["override_command_level"].get(name)
                    if level is None:
                        continue
                    else:
                        logger.info("Restored command permission level for `%s`.", name)
                        self.bot.config["override_command_level"].pop(name)
                        update = True

                if update:
                    await self.bot.config.update()
```

*Same as previous one.*